twine-upload.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+TWINE_USERNAME=$INPUT_USER TWINE_PASSWORD=$INPUT_PASSWORD twine upload dist/*
```

Comment:
If it's an entrypoint, shouldn't it `trap` signals on its own?

maybe
```suggestion
exec TWINE_USERNAME=$INPUT_USER TWINE_PASSWORD=$INPUT_PASSWORD twine upload dist/*
```